docs/tutorials/using_dpdk.md

```diff
@@ -0,0 +1,618 @@
+# Using DPDK
+
+In this tutorial, we will deploy User Plane Function (UPF) in DPDK mode using the [sdcore-user-plane-k8s] Terraform Module in a VM.
+[Multipass] is used to create User plane VM with a [LXD] backend.
+
+## Requirements
+
+A machine running Ubuntu 22.04 with the following resources:
+
+- At least one NIC with internet access
+- CPU that supports AVX2, RDRAND and PDPE1GB instructions (Intel Haswell, AMD Excavator or equivalent)
+- 8 cores
+- 16 GB RAM
+- 50 GiB disk
+
+## 1. Prepare the Host machine
+
+### Enable HugePages
+
+As a `root` user, update the Grub to enable 2 * 1Gi HugePages in the host machine. Then, the host is gracefully rebooted to activate the settings.
+
+```shell
+sudo sed -i "s/GRUB_CMDLINE_LINUX=.*/GRUB_CMDLINE_LINUX='default_hugepagesz=1G hugepages=2'/" /etc/default/grub
+sudo update-grub
+sudo init 6
+```
+
+#### Checkpoint 1: Are HugePages enabled ?
+
+You should be able to see the 2 * Free Hugepages with 1048576 kB size by executing the following command:
+
+```shell
+cat /proc/meminfo | grep Huge
+```
+
+The output should be similar to the following:
+
+```console
+AnonHugePages:         0 kB
+ShmemHugePages:        0 kB
+FileHugePages:         0 kB
+HugePages_Total:       2
+HugePages_Free:        2
+HugePages_Rsvd:        0
+HugePages_Surp:        0
+Hugepagesize:    1048576 kB
+Hugetlb:         2097152 kB
+```
+
+### Networks
+
+The following IP networks will be used for User Plane function deployment:
+
+| Name         | Subnet        | Gateway IP |
+|--------------|---------------|------------|
+| `management` | 10.201.0.0/24 | 10.201.0.1 |
+| `access`     | 10.202.0.0/24 | 10.202.0.1 |
+| `core`       | 10.203.0.0/24 | 10.203.0.1 |
+| `ran`        | 10.204.0.0/24 | 10.204.0.1 |
+
+On the host machine, create local network bridges to be used by LXD by adding below configuration under `/etc/netplan/99-sdcore-networks.yaml`.
```

Comment:
I'm not sure this howto should cover the creation of the network interfaces. Perhaps this should be its own howto?

Reply:
Whoops, sorry, I thought I was reading the howto PR, not the tutorial. Ignore this comment.